### PR TITLE
feat: add bestiary

### LINKED ISSUE #1461: API bestiary
image

bestiary object with info on kills and deaths

### PR DESCRIPTION
**Pull request by DuckySoLucky**
## Description 

Closes #1461 

## Examples
image
image
image

### CODE REVIEW

**Review by dukio — APPROVED** [review]
tested, lgtm!

**Comment by metalcupcake5** [comment]
is there a reason you're changing indentations?

**Comment by DuckySoLucky** [comment]
> is there a reason you're changing indentations?

I accidentally used [EJS Beautify](https://marketplace.visualstudio.com/items?itemName=j69.ejs-beautify)
I can revert it if it's required

**Comment by dukio** [comment]
> > is there a reason you're changing indentations?
> 
> I accidentally used [EJS Beautify](https://marketplace.visualstudio.com/items?itemName=j69.ejs-beautify) I can revert it if it's required

Oh so that's how... I thought you did it by hand 😂 
I've been looking for a formatter for EJS files that works... Prettier just gave up or was breaking stuff, this doesn't seem to be breaking anything right?

File: views/stats.ejs [hunk 1]
```diff
@@ -790,36 +832,35 @@ const metaDescription = getMetaDescription()
...
+            <span class="stat-name"><%= weight.name %>: </span>
+            <span class="stat-value"><%= parseFloat(Math.round(weight.total)).toLocaleString() %></span></span>
+        </div>
+        <%
           }
         %>
       </div>
 
       <div id="skill_levels_container">
         <% if ('levels' in calculated) { %>
```
Comment (metalcupcake5) [inline views/stats.ejs]: a lot of the indenting looks ok but if you could make statements like this have an indent that would be nice

**Comment by DuckySoLucky** [comment]
> > > is there a reason you're changing indentations?
> > 
> > 
> > I accidentally used [EJS Beautify](https://marketplace.visualstudio.com/items?itemName=j69.ejs-beautify) I can revert it if it's required
> 
> Oh so that's how... I thought you did it by hand 😂 I've been looking for a formatter for EJS files that works... Prettier just gave up or was breaking stuff, this doesn't seem to be breaking anything right?

It didn't break anything so I guess it works 🤷‍♂️

File: views/stats.ejs (hunk 1 above)
Comment (metalcupcake5) [reply views/stats.ejs]: >statements like this

meaning things that would be indented in javascript like `if`, `for`, etc

Comment (DuckySoLucky) [reply views/stats.ejs]: ~~You mean `<% if (statement) { %>` or?~~
nvm, didn't see second comment, GitHub didn't update

**Review by MartinNemi03 — APPROVED** [review]
lgtm, tested

**Comment by dukio** [comment]
@DuckySoLucky see https://github.com/DuckySoLucky/SkyCrypt/pull/7, once that change is done or my PR merged tag me so I can approve this PR :)

**Comment by DuckySoLucky** [comment]
> @DuckySoLucky see [DuckySoLucky#7](https://github.com/DuckySoLucky/SkyCrypt/pull/7), once that change is done or my PR merged tag me so I can approve this PR :)

Merged

**Review by dukio — APPROVED** [review]
lgtm

**Comment by dukio** [comment]
Small warning for devs, this now includes https://github.com/SkyCryptWebsite/SkyCrypt/pull/1701 as it was needed to fully debug the issue with this PR (will keep that PR open in case I missed something still, but shouldn't be the case)

**Pull request merged**